khard/config.py

```diff
@@ -92,7 +92,7 @@ def validate_private_objects(value):
 class Config:
 
     supported_vcard_versions = ("3.0", "4.0")
-    SPEC_FILE = os.path.join(os.path.dirname(__file__), 'data', 'config.spec')
+    SPEC_FILE = None
```

Comment:
This is never read so it should be removed, including the test if it is None below.